lib/chat_api/twilio.ex

```diff
@@ -82,6 +84,57 @@ defmodule ChatApi.Twilio do
     |> Repo.one()
   end
 
+  @spec find_or_create_customer_and_conversation(String.t(), String.t()) ::
+          {:ok, Customers.Customer.t(), Conversations.Conversation.t()}
+          | {:error, Ecto.Changeset.t()}
+  def find_or_create_customer_and_conversation(account_id, phone) do
```

Comment:
Im not sure if this should live here. I can see this being used for other integration from different sources.

Reply:
I agree, I think it should live in conversations (or customers) but probably not here